cybershell/scoring.py

```
@@ -0,0 +1,482 @@
+from dataclasses import dataclass
+import numpy as np
+from .plugins import PluginResult
+
+class BaseScorer(ABC):
+    """Base class for evidence scoring strategies"""
+    name: str = "base"
+
+    @abstractmethod
+    def score(self, result: PluginResult) -> float:
+        """Calculate evidence score from plugin result (0.0 to 1.0)"""
+        pass
```

Ruff (0.12.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)